lib/openstack_api/openstack_query.py

```diff
@@ -281,6 +285,11 @@ def get_user_prop(user_id, prop):
                 "vcpu_usage": lambda a: f"{a['vcpus_used']}/{a['vcpus']}",
                 "memory_mb_usage": lambda a: f"{a['memory_mb_used']}/{a['memory_mb']}",
                 "local_gb_usage": lambda a: f"{a['local_gb_used']}/{a['local_gb']}",
+                "uptime": lambda _: _raise(
```

Comment:
Do we need the stubbed raise?

I'm pretty sure `"uptime": lambda _: raise NotImplementedError(...)` would work as lambda and functions are first class citizens and therefore equal?

Reply:
I tried that initially but pycharm doesn't like it, seems its not valid https://stackoverflow.com/questions/8294618/define-a-lambda-expression-that-raises-an-exception